tests/all-tests.html

```diff
@@ -74,7 +74,7 @@
             ],
             "Testing Framework Tests": [
                 "/test-core/testTests/html/ConditionalTestUtils-test.html",
-                "/test-core/testTests/html/IoCTesting-test.html",
+//                "/test-core/testTests/html/IoCTesting-test.html",
```

Comment:
Shall this be uncommented?

Reply:
Ah, I saw your PR https://github.com/fluid-project/infusion/pull/1084 and now understand why `IoCTesting-test.html` is commented out. Did you figure out later why this test hangs on CI?

Reply:
Cheers, good catch - actually #1084 is already merged into this pull and so the test no longer hangs